docs/guides/products/products.md

```diff
@@ -0,0 +1,134 @@
+
+<ol className="numbered-list">
+  <li>
+    <span>
+      Navigate to your event website, then click "<strong>Housekeeping</strong>" from the left-hand sidebar.
+    </span>
+  </li>
+  <li>
+    <span>
+      Click "<strong>Products</strong>" from the left-hand sidebar, then click "<strong>View Products</strong>" next to the category you want to create the product in.
+    </span>
+  </li>
+  <li>
+    <span>Click "<strong>Create New Product</strong>".</span>
+  </li>
+  <li>
+    <div>
+      Enter basic information for your Product:
+      <ul>
+        <li>
+          <strong>Name</strong> - This is the name of your Product (like "T-Shirt" or "Bobblehead"). If you'd like to use a different name externally, you can do so in the "Display Name" field.
+        </li>
+        <li>
+          <strong>Description</strong> - A brief description of the product and its functionality.
+        </li>
+        <li>
+          <strong>Base Price</strong> - The price that will be charged for this product, before any add-ons or discounts are applied.
+        </li>
+        <li>
+          <strong>Stock Control</strong> - Whether or not you want to control the number of this product that can be purchased. This is useful for physical products where you have a fixed inventory available.
+        </li>
+        <li>
+          <strong>Pass Platform Fee to Customer</strong> - By default, our platform fees are absorbed into the cost of each product and are paid by the event, but as the organizer, you can choose to pass these fees to your attendees.
+        </li>
+        <li>
+          <strong>Make Product Publicly Visible</strong> - By default, your new product will not be shown in the Event Store, but you can make it visible by checking this box. This can be used for special products you don't want available for direct purchase, such as swag items for higher tier Attendance Types.
+        </li>
+      </ul>
+    </div>
+  </li>
+  <li>
+    <div>
+      <i>(Optional)</i> Customize advanced options:
+      <ul>
+        <li>
+          <strong>Available From</strong> - The date that this product will be available for purchase.
+        </li>
+        <li>
+          <strong>Available Until</strong> - The date that this product will no longer be available for purchase.
+        </li>
+        <li>
+          <strong>Minimum Purchase Age</strong> - The minimum age that attendees must be to purchase this product.
+        </li>
+        <li>
+          <strong>Maximum Purchase Age</strong> - The maximum age that attendees can be to purchase this product.
+        </li>
+      </ul>
+    </div>
+  </li>
+  <li>
+    <span>Click "<strong>Create Product</strong>", and your Product will be created!</span>
+  </li>
+  <li>
+    <div>
+      <i>(Optional)</i> Customize your Product:
+      <div style={{ marginBottom: "1rem" }} />
+      <splitColumn childStyle={{ width: "100%" }}>
+        <bigButton title="Product Images" href="#">
```

Comment:
These links when clicked cause the page to scroll to the top, this is probably not desired.

Reply:
Whoops, that needs to be fixed!